examples/oneview_logical_interconnect_group.yml

```diff
@@ -203,7 +328,7 @@
               mode: 'Auto'
               networkType: 'Ethernet'
               networkNames:
-                - 'TestNetwork_3'
+                - 'TestNetwork_5'
                   #                - '/rest/ethernet-networks/5cca6ddc-49b1-4c7c-9022-eadb2630d42b'
```

Comment:
remove this commented lines under NetworkNames

Reply:
Done